int8 count
astra_sdk_body[6] bodies